src/hipscat/io/parquet_metadata.py

```diff
@@ -30,7 +33,30 @@ def row_group_stat_single_value(row_group, stat_key: str):
     return min_val
 
 
-def write_parquet_metadata(catalog_path: str, storage_options: dict = None, output_path: str = None):
+def get_healpix_pixel_from_metadata(metadata) -> HealpixPixel:
+    """Get the healpix pixel according to a parquet file's metadata.
+
+    This is determined by the value of Norder and Npix in the table's data"""
+    if metadata.num_row_groups <= 0 or metadata.num_columns <= 0:
+        raise ValueError("metadata is for empty table")
+    order = -1
+    pixel = -1
+    first_row_group = metadata.row_group(0)
+    for i in range(0, first_row_group.num_columns):
+        column = first_row_group.column(i)
+        if column.path_in_schema == "Norder":
+            order = column.statistics.min
+        elif column.path_in_schema == "Npix":
+            pixel = column.statistics.min
+
+    if order == -1 or pixel == -1:
+        raise ValueError("Metadata missing Norder or Npix column")
+    return HealpixPixel(order, pixel)
+
+
+def write_parquet_metadata(
+    catalog_path: str, order_by_healpix=True, storage_options: dict = None, output_path: str = None
```

Comment:
Do we want the sky-ordering to be the default?

Reply:
Yes - we should have been doing this already for all _metadata files, except for `index` tables.